pkg/reaper/nodereaper/nodereaper.go

```diff
@@ -161,6 +161,14 @@ func (ctx *ReaperContext) validateArguments(args *Args) error {
 		ctx.ReapUnjoinedValue = args.ReapUnjoinedValue
 	}
 
+	if args.DrainTimeoutSeconds < 600 {
```

Comment:
should we set the default value to 600 if the argument is not set to maintain backward compatibility?

Reply:
Yeah.. also - is there a flag to control this? we should have a flag and the default value should be 600 seconds

Reply:
Sorry .. Somehow that files was missed from this PR. I have added it.